src/angular/styles/includes/_mixins.scss

```diff
@@ -164,6 +164,53 @@
   }
 }
 
+@mixin horizontalBorderAndShadowAsBoxShadow($attached: below, $borderWidth: 2, $scalingFactor: 1) {
```

Comment:
Is it intended to use horizontalBorderAndShadowAsBoxShadow and panelOffsetCorrection separately in isolation? If not, these could be merged into one mixin.

Reply:
From the technically perspective it could be merged, but I thought it's better to have them separated because its different functions. What do you think?